docker/docker-compose-base.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
+
+  mongo: # 27017
+    image: mongo:latest
+    network_mode: "host"
+    privileged: true
+
+  files:
+    build: "./files"
+    image: "rackhd/files"
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "./files/mount:/RackHD/files"
+
+  # RackHD services
+
+  dhcp-proxy: # 68/udp, 4011
+    build: "../on-dhcp-proxy"
+    depends_on:
+      - mongo
+    image: rackhd/on-dhcp-proxy:latest
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "dhcp-leases:/var/lib/dhcp"
+      - "./monorail:/opt/monorail"
+
+  http: # 9090, 9080
```

Comment:
I would think this should also depends on core service (https://github.com/RackHD/RackHD/blob/master/docker/docker-compose.yml#L47) and tasks(https://github.com/RackHD/RackHD/blob/master/docker/docker-compose.yml#L52) service, but I believe this file is tested to prove that core and tasks services are not needed.